game.py

```diff
@@ -20,7 +20,38 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    
+    snowman_word_dic = build_letter_status_dict(snowman_word)
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_count = 0
```

Comment:
It can sometimes be useful to have a counter to keep track of how many items we have. One thing to remember however is that most collections (lists, sets, dictionaries, etc...) have a built in attribute that keeps track of how many items are in it. To access that attribute, we just have to use the `len()` function. Because access is so easy, it's good practice to just use the len() function as opposed to duplicating that information with your own variable.